src/main/resources/templates/fragments/fileUploader.html

```diff
@@ -386,7 +395,8 @@
               name: doc.originalFilename,
               size: doc.filesize,
               type: doc.type,
-              id: parseInt(uploadedDocWithThumbnail[0])
+              id: parseInt(uploadedDocWithThumbnail[0]),
+              accepted: true
```

Comment:
This fixes the issue of dropzone not noticing that we hit the max file limit if we had the max files, clicked continue and then went back.  

Dropzone has the concept of "accepted" files, so if they were not accepted, they didn't count towards maxFiles.